frontend/src/components/NavBar.tsx

```diff
@@ -30,23 +31,44 @@ const NavBar = () => {
           isDarkMode ? "navbar-dark bg-dark" : "navbar-light bg-light"
         }`}
       >
-        <div className="container-fluid">
+        <div className="container-fluid d-flex justify-content-between align-items-center">
           <Link className="navbar-brand me-4 py-1" to="/">
-            <img src={logo} alt="Bowtie" width="128px" />
+            <img src={logo} alt="Bowtie Logo" width="128px" />
           </Link>
-          <button
-            className="navbar-toggler"
-            type="button"
-            data-bs-toggle="collapse"
-            data-bs-target="#navbarNav"
-            aria-controls="navbarSupportedContent"
-            aria-expanded="false"
-            aria-label="Toggle navigation"
-          >
-            <span className="navbar-toggler-icon"></span>
-          </button>
+          <div className="d-flex align-items-center">
+            <div className="d-lg-none">
+              <Button className="btn btn-secondary border-0 me-1">
+                <Link
+                  className="nav-link"
+                  to="https://docs.bowtie.report/"
```

Comment:
Clicking this link doesn't appear to work, possibly because you have `target="blank"` below (as opposed to `_blank`), but there's no reason for `_blank` either, you can just remove that entirely.

Reply:
Oh, it seems `target="blank"` is present in the current version, and that one works. So not sure what the reason is, but yeah have a look.

Reply:
Oh very strange, the behavior is even weirder, I'm not sure what's happening. Here it works when clicked the first time, and not after that:


https://github.com/bowtie-json-schema/bowtie/assets/329822/49a613dc-991b-4b90-bdbf-804320f154de

When nothing is happening I'm still mashing clicking the button, but nothing is happening. I'd try fixing the `target` to `_blank`, or removing it. But let me know if you can reproduce at all.

Reply:
Thanks a lot, It was tricky to figure out the problem, why is this doing weird behavior. Now every thing works fine, as expected

![Frame 2 (4)](https://github.com/bowtie-json-schema/bowtie/assets/92979541/1443ab15-1824-477e-9eb9-e146e36a8cc7)

## Why It was doing weird behavior

- As you can See in the above Image Link was only in the text "Docs". Icon was outside of the link. that's why this was doing weird behavior.
- Also I used the link inside of a `button`. That's why when you clicked on the white space like below image, The Link wasn't creating a new window

![Frame 2 (5)](https://github.com/bowtie-json-schema/bowtie/assets/92979541/34b5be83-588d-48ab-b403-40fac773b01f)

### Also I have learned a new thing today
![image](https://github.com/bowtie-json-schema/bowtie/assets/92979541/848a745f-61d3-43c5-a6a6-3670091398ce)

- If you want to know more you can pay a visit to https://css-tricks.com/targetblank/

# Is there anything I can do?

Reply:
Very interesting, well done, thanks for sharing and laying things out clearly.